platform/go/go-std/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "target": {
+    "path": "/update?queries=10"
+  },
+  "rps": 100
```

Comment:
@AbhiPrasad this was "easy" to run at 100 RPS, vs Python and Node struggling to handle 10 RPS.
Seems good to have the knobs per-platform/framework.

Reply:
Nice - I wonder if there is an easy programmatic way to test what rps to use.

Reply:
There's definitely a way. For example, pick two extremes for RPS, `min` and `max`, and do a binary search to find the `max'` RPS value that can be used to complete the test run without errors (a la `git bisect`).

The `max'` value will be different depending on hardware and possibly other external influences.

I don't think it is worth diving into that right now :)